docs/DeveloperGuide/Layer/Usecase/TrafficSimulation/Instruction/index.md

```diff
@@ -0,0 +1,132 @@
+`LaneletLoader` can be performed by opening `AWSIM -> Random Traffic -> Load Lanelet` at the toolbar of Unity Editor.
+
+![Tool Bar](./load_lanelet_tool_bar.png)
+
+Please use `LaneletLoader` as the following:
+1. Fill in `Osm` field with a lanelet map (`.osm`) you prepared, `Root Object` field with a `TrafficSimulator` object.<br>
+2. Adjust `Waypoint settings` parameters for the loading process if needed.<br>
+3. To load the lanelet map, please click `Load` button.
+
+![Load Lanelet](./load_lanelet.png)
+
+The `Waypoint settings` parameters are listed in the following table:
+
+| Parameter | Description |
+|---|---|
+| Resolution | Resolution of resampling. Lower values provide better accuracy at the cost of processing time |
+| Min Delta Length | Minimum length(m) between adjacent points |
+| Min Delta Angle | Minimum angle(deg) between adjacent edges<br>Lowering this value produces a smoother curve |
+
+`TrafficLane`, `StopLine` and `TrafficLight` will be generated and placed as child objects of the `Root Object`.<br>
+You can check their visual representation by clicking consecutive elements in the scene hierarchy.
+
+![Hierarchy](./hierarchy2.png)
+
+## TrafficSimulator setting
+
+![Traffic Simulator](./traffic_simulator.png)
+
+Please configure `TrafficSimulator` component as the following:
+
+1. Fill in `Ego Vehicle` field with EGO vehicle
+    1. Example: `EgoVehicle/Lexus RX450h 2015`
+2. Change `Obstacle Layer Mask` and `Ground Layer Mask` field to `Everything`
+3. Fill in `Traffic Intersections` field with `TrafficIntersection` objects
+4. Fill in `Random Traffic Sims` field
+    1. Fill in `Traffic Sim Npc Vehicle Prefab` field with vehicle prefabs what you want to spawn
+        1. Prefabs is in `Assets/Awsim/Prefabs/Usecase/TrafficSimulation/`
+    2. Fill in `Spawnable Traffic Lanes` field with `TrafficLane` where you want to spawn vehicles
+
+For detailed settings, see [here](../Abstract/index.md#configulations)
+
+## Place Pedestrian (optional)
+You can place pedestrian NPCs if needed.<br>
+Pedestrians can animated and walk around where they are placed.<br>
+Direction which pedestrian start to walking can be set in `Transform` of it.
+
+![Pedestrian](./pedestrian.png)
+
+Please configure pedestrian NPCs as the following:
+
+1. Place pedestrian prefabs on a scene
+    1. Prefabs is in `Assets/Awsim/Prefabs/Entity/Npc/Pedestrian/`
+    2. Pedestrians should be child object of `NPCPedestrians` hierarchy
+2. Attach `SimplePedestrianWalkerController` component to pedestrians
+3. (optional) Configure parameters of `SimplePedestrianWalkerController`
+
+The parameters of `SimplePedestrianWalkerController` are listed in the following table:
+
+| Parameter | Description |
+|---|---|
+| Duration | Duration at which the pedestrian walk |
+| Speed | Speed at which the pedestian walk |
+
+## Reference Components
+To enable `Traffic Simulation`, please fill in following fields in `AutowareSimulationDemo.cs`.
```

Comment:
Do not write this document in a way that depends on `AutowareSimulationDemo.cs`. It's fine to open the `AutowareSimulationDemo` scene as a sample for reference. (An info box would be appropriate.) 
The functions in the Use Case layer are intended to be used generically, not limited to specific scenarios.

This section should primarily describe how TrafficSimulator.Initialize() and TrafficSimulator.OnUpdate() need to be called.